src/main/resources/lib/guillotine/index.js

```diff
@@ -65,10 +67,93 @@ function createContext(options) {
     }
     context.options.applications = context.options.applications || [app.name];
     context.options.allowPaths = context.options.allowPaths || [];
+    context.options.subscriptionEventTypes = context.options.subscriptionEventTypes || ['node.*'];
 
     return context;
 }
 
+//──────────────────────────────────────────────────────────────────────────────
+// GraphQL WS Protocol
```

Comment:
Move to own file - in the lib-grphql?

Reply:
@rymsha I guess will be better to figure out the better solution for that issue and then move it to lib-graphQL if needed, but not now.